troposphere/static/js/actions/QuotaActions.js

```diff
@@ -0,0 +1,45 @@
+define(function (require) {
+
+  var Utils = require('./Utils');
+      Router = require('../Router'),
+      Constants = require('constants/QuotaRequestConstants'),
+      QuotaRequest = require('models/QuotaRequest');
+
+  return {
+      //create: function (params) {
```

Comment:
I think that can be removed since creation is handled through HelpActions.js and we don't actually delete requests from the database we just change the status.